packages/cli/src/bin/portal-upload.ts

```diff
@@ -141,6 +143,18 @@ export const uploadPortalBundle = async (opt: UploadPortalOptions) => {
     // Log and return the successful response
     log?.succeed('Successfully uploaded portal bundle');
     log?.debug('Response:', response);
+
+    try {
+      const result = await (response.json() as Promise<{ version: string }>);
```

Comment:
will this work? @odinr @alftore

Reply:
dunno, have you tested it?

but if the server responded with ok, isnt that good enough 🤷

Reply:
can I use version and name form the manifest in options?